Unreal/DazStudioPlugin/DzUnrealAction.h

```diff
@@ -11,15 +11,17 @@
 #include "DzUnrealSubdivisionDialog.h"
 
 class DzUnrealAction : public DzRuntimePluginAction {
-    Q_OBJECT
+	Q_OBJECT
     Q_PROPERTY(int NonInteractiveMode READ getNonInteractiveMode WRITE setNonInteractiveMode)
+	Q_PROPERTY(QString AssetType READ getAssetType WRITE setAssetType)
 public:
 	 DzUnrealAction();
 
 protected:
 	 int Port;
      int NonInteractiveMode;
 	 DzUnrealSubdivisionDialog* SubdivisionDialog;
+	 QString AssetType;
```

Comment:
**Correction:**
Solution provided above is incomplete, since the value of DzRuntimePluginAction::AssetType is over-written by the value within the GUI combobox for AssetType.  The script-exposed AssetType property can not be directly linked to the GUI combobox value since the GUI is created dynamically and may potentially be NULL at any point in time.  Fully correct solution would be to use signal/slot-based GUI validation system to enforce consistency of DzRuntimePluginAction::AssetType at the time of combobox creation.

Reply:
AssetType moved to DzRuntimePluginAction with latest commit.  Script overrides UI and stubs for future GUI to Script integration also implemented.